gradient_engine.py

```
@@ -0,0 +1,189 @@
+            # Broadcast to known peers
+            for peer_host, peer_port in self.peers.copy():
+                await self.exhale(peer_host, peer_port, payload)
+
+            print(f"[GRADIENT] Coherence: {self.engine.calculate_coherence():.3f} | Forecast: {self.engine.forecast_risk(24)}")
+
+            await asyncio.sleep(self.breath_interval)
+
+    async def start(self):
+        """Start the guardian with breathing cycle"""
+        # Start the breathing cycle
+        asyncio.create_task(self.breath_cycle())
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)